lib/internal/main/run_main_module.js

```diff
@@ -10,8 +10,4 @@ const CJSModule = require('internal/modules/cjs/loader').Module;
 
 markBootstrapComplete();
 
-// Note: this actually tries to run the module as a ESM first if
```

Comment:
I think the comment here is still somewhat valid and should stay (with a tweak) - this actually tries to run the module as a ESM if we somehow know that the module should be treated as ESM. That's surprising when you read this considering `const CJSModule = require('internal/modules/cjs/loader').Module;`.